spec/system/support_interface/reasons_for_rejection_sub_reasons_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.feature 'Reasons for rejection sub-reasons' do
+  include DfESignInHelpers
+
+  around do |example|
+    @today = Time.zone.local(2020, 12, 24, 12)
+    Timecop.freeze(@today) do
+      example.run
+    end
+  end
+
+  scenario 'View reasons for rejection sub-reasons' do
+    given_i_am_a_support_user
+    and_there_are_candidates_and_rejected_applications_in_the_system
+
+    when_i_visit_the_reasons_for_rejection_sub_reasons_page
```

Comment:
We need to add a tab to the performance page that links to this page. Maybe we could add clicking it to this test?